packages/tools/devtools/devtools-view/src/test/jest/tsconfig.esm.json

```diff
@@ -0,0 +1,18 @@
+{
+	// This config must be used in a "type": "module" environment. (Use fluid-tsc module.)
+	"extends": "../../../../../../../common/build/build-common/tsconfig.test.node16.json",
+	"include": ["./**/*"],
+	"compilerOptions": {
+		"rootDir": "./",
+		"outDir": "../../../lib/test/jest",
+		"types": [],
```

Comment:
Shouldn't this and its sibling file have "jest" types here, like the previous tsconfig?

Reply:
It should. jest typing appears to be transitively picked up with import of @testing-library/jest-dom, but we should be explicit. I'll patch.